.gitlab-ci.yml

```diff
@@ -137,5 +147,28 @@ test:behat:cleanup:
   only:
   - merge_requests
   - master
+  except:
+  - schedules
   dependencies:
   - test:behat
+
+schedule:composer:update:
+  # Temporarily specify explicit image for testing. Can be removed when https://github.com/pantheon-systems/docker-build-tools-ci/pull/30 is merged.
+  image: quay.io/pantheon-public/build-tools-ci:clu-0-6-0
+  stage: build
```

Comment:
`build` is an invalid stage. `configure-and-build` is a valid stage but perhaps we should add a new stage `updates` that could be used for this job as well as other updates in the future.